contributors/devel/faster_reviews.md

```diff
@@ -1,3 +1,4 @@
-# How to get faster PR reviews
-
-Merged with the [pull requests doc](/contributors/guide/pull-requests.md#best-practices-for-faster-reviews).
+The contents of this file have been moved to  https://git.k8s.io/community/contributors/guide/pull-requests.md.
+ <!--
+ This file is a placeholder to preserve links.  Please remove after 3 months or the release of kubernetes 1.10, whichever comes first.
```

Comment:
I'm still unclear on the policy for how long to keep these tombstones, but as we get closer to 1.10 release, we probably should think about changing 3 mo's or 1.10 to instead be 3 mo's or 1.11.